dipy/tracking/utils.py

```diff
@@ -413,16 +413,15 @@ def seeds_from_mask(mask, density=[1, 1, 1], voxel_size=None, affine=None):
 
 
 def random_seeds_from_mask(mask, seeds_count=1, seed_count_per_voxel=True,
-                           affine=None):
+                           affine=None, random_seed=0):
```

Comment:
@GuillaumeTh Could you test how complicated it would be to also manage the case where `random_seed=None`, i.e. set to the previous default behavior? In case someone wants to keep the original behavior.